tests/parm/postxconfig-NT.txt

```diff
@@ -1,6 +1,6 @@
 2
-112
-197
+47
```

Comment:
@ericaligo-NOAA I would suggest you use parm/postxconfig-NT-GFS-TWO.txt from the UPP develop branch to update this file.